app/models/user.rb

```diff
@@ -0,0 +1,2 @@
+class User < ApplicationRecord
```

Comment:
- [ ] Kindly set up your model properly by adding validation on each of the users' table columns.
It should validate the username, email, and password fields to prevent bad user intentions. 🙏 😄
![user_model_attr_validation](https://user-images.githubusercontent.com/47625407/128914395-7c4a85a0-7f5e-462f-b76c-f955a4793d82.png)